### DISCUSSION TITLE
q-table with vuex store array as row prop

### DISCUSSION THREAD

**Discussion by dbarrow** [quasarframework/quasar#10429] [General - Components / Directives / etc]
I am setting the rows of a qtable with an array from a vuex store. But if I set the records per page to anything larger than the length of rows, when a column is sorted, a duplicate row is added to the table.

A hard coded array of data fed to the table does not produce this problem.

Vue dev tools does not show the duplicate row added to the store.

Any ideas?

Thanks

**Comment by ontwikkelfabriek** [comment]
@dbarrow 

Create a codepen.io to get better support for your problem.

**Comment by dbarrow** [comment]
In an effort to put together a codesandbox, I converted the API from local to live server.  

From the live server API, the table sorting behaves properly.  Switched back to local API and duplicate rows occur.  Weird!
 
The live and local API are identical in terms of source code.  Server configs are different.  

So, I cant provide a working example online.  If anyone would like to see a video, let me know.

I will update if I find the culprit.

Thanks
